src/app/topic/topic.component.html

```diff
@@ -11,39 +11,41 @@
     }">Topic</ion-title>
   </ion-toolbar>
 </ion-header>
-<ion-content color="light" [ngClass]="{'ion-content-absolute-with-footer': !utils.isMobile()}">
+<ion-content padding color="light" [ngClass]="{'ion-content-absolute-with-footer': !utils.isMobile()}">
```

Comment:
@sasangachathumal remember to use `class="ion-padding"` instead of `padding` attribute